addons/kubelet-configmap/kubelet-configmap.yaml

```diff
@@ -375,3 +375,94 @@ data:
     - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
     - TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305
     volumeStatsAggPeriod: 1m0s
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubelet-config-1.24
```

Comment:
Please note that this has been changed. `kubelet-config` ConfigMaps are no longer versioned (there's no `x.yy` suffix).

Changelog has some more details about this: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#deprecation

Reply:
I missed that, thank you. Not sure how to gracefully handle this but I will take a look.